src/snippets/responsive-bg-image.liquid

```diff
@@ -0,0 +1,36 @@
+{% comment %}
+    It creates a style tag containing "background-image" properties to load the correct image depending on the resolution
+    Dependencies:
+    - Lazysizes plugin (https://github.com/aFarkas/lazysizes) which enable responsive image with faster load time and better performance.
+    - Lazysizes Responsive Images as a Service (https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/rias) To load the correct image size with our CDN
+    - Lazysizes Bgset (https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/bgset) To use responsive images on background-image (CSS)
+
+    Accepts:
+    - image: {Object} Image object
+
+    Usage:
+    In your liquid template file, copy the following line
+
+    <div class="responsive-bg-image-wrapper">
+      <div class="responsive-bg-image lazyload" {% include 'responsive-bg-image', image: article.image %}></div>
+    </div>
+
+    - It must include parent DIV with "responsive-bg-image-wrapper" class and "responsive-bg-image lazyload" in the children element
+
+    Questions:
+    1- If you check the implementation in article.liquid, what do you think about the implementations?
+    2- Is it understandable and convenient to use include "responsive-bg-image"?
+    3- Since we need a few CSS classes, do you think I should put the <div> in this file also?
+{% endcomment %}
+{%- if image != blank -%}
+  data-bgset="
+    {% if image.width > 540 %}{{ image | img_url: '540x' }} 540w {{ 540 | divided_by: image.aspect_ratio | round }}h,{% endif %}
+    {% if image.width > 720 %}{{ image | img_url: '720x' }} 720w {{ 720 | divided_by: image.aspect_ratio | round }}h,{% endif %}
+    {% if image.width > 900 %}{{ image | img_url: '900x' }} 900w {{ 900 | divided_by: image.aspect_ratio | round }}h,{% endif %}
+    {% if image.width > 1080 %}{{ image | img_url: '1080x' }} 1080w {{ 1080 | divided_by: image.aspect_ratio | round }}h,{% endif %}
+    {% if image.width > 1296 %}{{ image | img_url: '1296x' }} 1296w {{ 1296 | divided_by: image.aspect_ratio | round }}h,{% endif %}
+    {% if image.width > 1512 %}{{ image | img_url: '1512x' }} 1512w {{ 1512 | divided_by: image.aspect_ratio | round }}h,{% endif %}
+    {% if image.width > 1728 %}{{ image | img_url: '1728x' }} 1728w {{ 1728 | divided_by: image.aspect_ratio | round }}h,{% endif %}
```

Comment:
Let's add some more breakpoints here? The array in the [lazysizes docs](https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/rias#list-of-options) is `[180, 360, 540, 720, 900, 1080, 1296, 1512, 1728, 1944, 2160, 2376, 2592, 2808, 3024]`